internal/cmd/run/run.go

```diff
@@ -142,6 +143,7 @@ func Command() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&gFlags.selectedSuite, "select-suite", "", "Run specified test suite.")
 	cmd.PersistentFlags().BoolVar(&gFlags.testEnvSilent, "test-env-silent", false, "Skips the test environment announcement.")
 	cmd.PersistentFlags().BoolVar(&gFlags.disableUsageMetrics, "disable-usage-metrics", false, "Disable usage metrics collection.")
+	cmd.PersistentFlags().BoolVar(&gFlags.disableCIInfo, "disable-ci-info", false, "Disable CI info collection.")
```

Comment:
I'd suggest making this flag more generic. We are auto tagging the jobs. CI information happens to be one of the infos we collect on behalf of the user. How about `--no-auto-tagging`?

Reply:
sounds better!